src/models/accounts.js

```diff
@@ -8,15 +8,93 @@ const db = require('./db/accounts')
 const hashPassword = require('five-bells-shared/utils/hashPassword')
 const NotFoundError = require('five-bells-shared/errors/not-found-error')
 const UnauthorizedError = require('five-bells-shared/errors/unauthorized-error')
+const InvalidBodyError = require('five-bells-shared/errors/invalid-body-error')
+const uri = require('../services/uriManager')
+const validator = require('../services/validator')
+
+function getPublicData (data) {
```

Comment:
There's no need for that because the exports object enforces that it is private to this file